src/NServiceBus.NHibernate/TimeoutPersisters/NHibernateTimeoutStorage.cs

```diff
@@ -54,10 +56,14 @@ protected override void Setup(FeatureConfigurationContext context)
                 }
             }
 
+            var timeoutsCleanupExecutionInterval = 
+                context.Settings.GetOrDefault<TimeSpan?>("NHibernate.Timeouts.CleanupExecutionInterval") ?? TimeSpan.FromMinutes(2);
```

Comment:
Good point. Here are my arguments:
 * When it comes to default override we had a discussion with @andreasohlund and @timbussmann and decided that we will not expose the setting through api method for now. Thanks to that we: a. do not have to pump package major number to follow semver  and b. if any user will want to change that we can still give the guidance on how to do it. This change is very similar to RavenDB approach where far none of the users requested the ability to change the default.
 * When it comes to clean-up period once 2 min the arguments (not scientific for sure) are: 
  * this is value for RavenDB [QueryTimeouts](https://github.com/Particular/NServiceBus.RavenDB/blob/develop/src/NServiceBus.RavenDB/Timeouts/QueryTimeouts.cs#L19)
   * ~3 (2min from persister + 1min from poller) min is now worse case scenario time for which the deferred will be delayed. I do know it sounds pretty arbitrary but I would argue it should be a couple of minutes and probably no more than 10.